modules/ROOT/pages/apikit-4-scaffolding-reference.adoc

```diff
@@ -0,0 +1,239 @@
+= Scaffolding Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: creating-an-apikit-4-project-with-maven.adoc
+
+Generate the backbone of a REST API implementation based on RESTful API Modeling Language (RAML) or OpenAPI Specifications (OAS) with APIkit for REST scaffolder.
+
+== Tooling
+
+Studio offers several ways to get an API scaffolded:
```

Comment:
"scaffold an API"?

Reply:
I'll try to reword keeping the coherence with the other H2.
Thanks!

Reply:
I meant to suggest replacing "get an API scaffolded" with "scaffold an API" (to avoid passive voice and "get").

Reply:
@Crispy-Salesforce

Reply:
Oh! I didn't get that! You're right!
Thanks!